Add Last Block Size to the Debug UI

ptschip opened the pull request:
This shows us the size of the last block processed and accepted
  at the chain tip.  It does not currently update when rolling back
  the chain but will as the chain re-org has finally completed.

Remove Client Name from the debug window to save space

ptschip committed:
Add Last Block Size to the Debug UI
  This shows us the size of the last block processed and accepted
  at the chain tip.  It does not currently update when rolling back
  the chain but will as the chain re-org has finally completed.

Remove Client Name from the debug window to save space

gandrewstone commented on src/requestManager.h:
```diff
@@ -160,6 +160,9 @@ class CRequestManager
 
+    // Size of last block that was successfully connected at the tip.
+    std::atomic<uint64_t> nLastBlockSize;
+
```
I feel like this should not be in the requestManager, since it doesn't have anything to do with handling tx or block requests.  What about using a global variable?

ptschip replied:
ok done

ptschip committed:
Make nLastBlockSize global and change to name nBlockSizeAtChainTip

This removes it from the request manager which should be use only
for requesting objects and also clarifies it's function by renaming.
Furthermore, nLastBlockSize is already a global and is used in miner.cpp
to track the last block size mined.
